### DISCUSSION TITLE
resticprofile web-based UI

### DISCUSSION THREAD

**Discussion by militu** [creativeprojects/resticprofile#401] [Show and tell]
Hello everyone,

I wanted to share a small project I've been working on for resticprofile. It's a web-based UI for exploring resticprofile configs and backups across different servers. Nothing too fancy, just something I put together to solve a personal need.

If you're interested, you can find it here: [resticprofile-ui](https://github.com/militu/resticprofile-ui)

It's still a work in progress, and there's certainly room for improvement. If anyone wants to take a look or has any feedback, I'd appreciate it.

Thanks again to the resticprofile maintainers for creating such a useful tool! Much appreciated!

Best regards,
Militu

**Comment by creativeprojects** [comment]
Hey!

Sorry I was away the last few weeks.

That's really cool 😎  I'll have a look and play with it 😉 

Thanks!

**Reply by militu** [reply]
Great! Thanks!
